cutlass/wmma_matrix.h

```diff
@@ -30,20 +30,20 @@
 #if defined(__CUDACC__) && (!defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 700)
 #define CUTLASS_USE_WMMA_API
 
-#if defined(__CUDACC__) && (__CUDACC_VER_MAJOR__ >= 10) && (!defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 720)
+#if defined(__CUDACC__) && (CUDA_VERSION >= 10000) && (!defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 720)
```

Comment:
I see a few places the CUDA_VERSION is bumped to 10000. Can you elaborate the reasoning behind this? We would like to be able to define `CUTLASS_USE_INT_WMMA` in this case if CUDA_VERSION is larger than 10, for example.

Reply:
I did *not* bump the version -- it was 10 in the original code.

I've replaced `__CUDACC_VER_MAJOR__` which is not set by clang to `CUDA_VERSION` which matches the intent of this code and is available to both nvcc and clang.

Reply:
ah I see the difference now. Thanks for clarification.